<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutDialog</class>
 <widget class="QDialog" name="AboutDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>707</width>
    <height>381</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>707</width>
    <height>363</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>707</width>
    <height>381</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>About Openify</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>707</width>
     <height>111</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>707</width>
     <height>111</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>707</width>
     <height>111</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/banner/pictures/banner_about.png);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
  </widget>
  <widget class="QLabel" name="version">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>87</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string>Client version: 0.0.0.0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="server_version">
   <property name="geometry">
    <rect>
     <x>541</x>
     <y>65</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>661</width>
     <height>221</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="styleSheet">
       <string notr="true">font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Client</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="system_info">
      <property name="text">
       <string>system_info</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="compatible_controller_version">
      <property name="text">
       <string>compatible_controller_version</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="styleSheet">
       <string notr="true">font-weight: bold;</string>
      </property>
      <property name="text">
       <string>Server</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="arch">
      <property name="text">
       <string>Retreiving...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="os">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="go_version">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="min_controller_version">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="current_controller_version">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
